app/src/lib/editors/launch.ts

```diff
@@ -18,7 +18,12 @@ export async function launchExternalEditor(
 
   //Special launching for Xcode.
   if (editor.editor === Darwin.Xcode) {
-    //Query the file system for xCode related files
+    //Directly open if the fullPath contains a file
+    if (lstatSync(fullPath).isFile()) {
```

Comment:
Nit: could we use the async `await lstat` please? I don't think there should be a reason to use sync operations in an already `async` function.

Reply:
Sure. Do you think this change will work?

`if (await lstat(fullPath).isFile())`

Will push the new commit today

Reply:
Changes pushed @mislav